components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -1728,22 +1752,22 @@ subroutine get_cloud_dsd2(qc,nc,mu_c,rho,nu,dnu,lamc,lammin,lammax,cdist,cdist1,
        endif
 
        ! calculate lamc
-       lamc = (cons1*nc*(mu_c+3._rtype)*(mu_c+2._rtype)*(mu_c+1._rtype)/qc)**thrd
+       lamc = bfb_cbrt(cons1*nc*(mu_c+3._rtype)*(mu_c+2._rtype)*(mu_c+1._rtype)/qc)
 
        ! apply lambda limiters
        lammin = (mu_c+1._rtype)*2.5e+4_rtype   ! min: 40 micron mean diameter
        lammax = (mu_c+1._rtype)*1.e+6_rtype    ! max:  1 micron mean diameter
 
        if (lamc.lt.lammin) then
           lamc = lammin
-          nc   = 6._rtype*lamc**3*qc/(pi*rhow*(mu_c+3._rtype)*(mu_c+2._rtype)*(mu_c+1._rtype))
+          nc   = 6._rtype*(lamc*lamc*lamc)*qc/(pi*rhow*(mu_c+3._rtype)*(mu_c+2._rtype)*(mu_c+1._rtype))
        elseif (lamc.gt.lammax) then
           lamc = lammax
-          nc   = 6._rtype*lamc**3*qc/(pi*rhow*(mu_c+3._rtype)*(mu_c+2._rtype)*(mu_c+1._rtype))
+          nc   = 6._rtype*(lamc*lamc*lamc)*qc/(pi*rhow*(mu_c+3._rtype)*(mu_c+2._rtype)*(mu_c+1._rtype))
```

Comment:
So do you in fact need to do `lamc*lamc*lamc`? I had thought, I guess mistakenly, that `lamc**3`, ie with integer 3, expands to that.

Reply:
Ideally, we would have `lamc**3` in F90 and `cube(lamc)` in C++. I don't want to make you have to rerun tests for this PR, but I recommend you make these mods in your next PR.